index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="index.css">
+    <title>Document</title>
+</head>
+<body>  
+    <header class="myheader">
+        <h1>Hello Microverse!</h1>
+    </header>
```

Comment:
**[Optional]**

- [ ] You should have give class directly to your `h1` to change your h1 text color :thinking: In this case, you are not changing the Hello Microverse Color. You can see following to get the idea what i am talking about :+1:

**Your Solution :red_circle:**
```

<header class="myheader">
  <h1>Hello Microverse!</h1>
</header>
```
**Good Solution :green_circle:**
```
<header>
  <h1 class="myheader">Hello Microverse!</h1>
</header>
```